torchvision/transforms/functional.py

```diff
@@ -454,8 +455,12 @@ def resize(
     if not torch.jit.is_scripting() and not torch.jit.is_tracing():
         _log_api_usage_once(resize)
 
-    if not isinstance(interpolation, InterpolationMode):
-        raise TypeError("Argument interpolation should be a InterpolationMode")
+    if isinstance(interpolation, int):
+        interpolation = _interpolation_modes_from_int(interpolation)
+    elif not isinstance(interpolation, InterpolationMode):
+        raise TypeError(
+            "Argument interpolation should be a InterpolationMode or an corresponding Pillow integer constant"
```

Comment:
(same in similar error messages)

```suggestion
            "Argument interpolation should be a InterpolationMode or a corresponding Pillow integer constant"
            f"Got {interpolation} of type {type(interpolation).}"
```